src/test/test-honor-first-shutdown.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo "Honor first shutdown test script"
+count=1
+while [ $count -eq 1 ]; do
```

Comment:
What is the purpose of the $count variable? What is the purpose of the loop?

Reply:
Will remove the count. The loop provides a simulated main pid that can get killed by execstop to generate the FailureAction=reboot action during a poweroff initiated by the end.service in the added tests.